### DISCUSSION TITLE
Search toggle not working

### DISCUSSION THREAD

**Discussion by changyaochen** [mmistakes/minimal-mistakes#3788] [Q&A] [answered]
Hi community

I have created a personal blog site using minimal-mistake, it works perfectly. I then want to add the search functionality, however, the search toggle doesn't seem to do anything... I spent about a day, only to no avail...

Link to the [site](https://changyaochen.github.io/).
Link to the [repo](https://github.com/changyaochen/changyaochen.github.io).

Any help would be highly appreciated!

**Comment by iBug** [comment, marked as answer]
If you're not familiar with web development in general, you should use [the starter repo](https://github.com/mmistakes/mm-github-pages-starter) instead of forking the theme.

Precisely, this is what you're doing wrong:

<https://github.com/changyaochen/changyaochen.github.io/blob/444750df3f2593fd3c660d721b7df143b0854469/_includes/scripts.html#L6>

**Reply by changyaochen** [reply]
Thanks for digging into the details! Highly appreciated! 

That is indeed the culprit!
